src/renderer/components/jotz.js

```diff
@@ -3,48 +3,69 @@ var SideMenu = require('./side_menu/side_menu');
 var Content = require('./content/content');
 var TopBar = require('./top_bar/top_bar');
 var actionCreator = require('../actions/action_creator');
+//var NotesStore = require('../stores/notes');
+//var NotebookStore = require('../stores/notebooks');
 
 /*
   This is where the applications state is created/managed.
   TODO: Link up fetchNotes to allNotes for notelist view (getInitialState)
-  TODO: Decide on React.js style guide eg: changeView vs newNote
  */
 
 var Jotz = React.createClass({
 
-  //State created here
   getInitialState: function() {
     return {
+      notes: null,
+      notebooks: null,
       view: 'Notes',
       filterQuery: ''
     };
   },
 
-  updateSearch: function (event) {
-    this.setState({
-      filterQuery: event.target.value
+  componentDidMount: function() {
+    actionCreator.fetchNotes();
+    var updateStores = function() {
+      var newState = React.addons.update(this.state, {
+        notes: { $set: this.props.notes.toJSON() },
```

Comment:
Why would we need notes and notebooks as props. Why can't we just leave as state and set it in this jotz file?